internal/service/backupcompliancepolicy/resource_backup_compliance_policy.go

```diff
@@ -239,10 +238,10 @@ func resourceCreate(ctx context.Context, d *schema.ResourceData, meta any) diag.
 		AuthorizedEmail:         d.Get("authorized_email").(string),
 		AuthorizedUserFirstName: d.Get("authorized_user_first_name").(string),
 		AuthorizedUserLastName:  d.Get("authorized_user_last_name").(string),
-		CopyProtectionEnabled:   pointy.Bool(d.Get("copy_protection_enabled").(bool)),
-		EncryptionAtRestEnabled: pointy.Bool(d.Get("encryption_at_rest_enabled").(bool)),
-		PitEnabled:              pointy.Bool(d.Get("pit_enabled").(bool)),
-		RestoreWindowDays:       pointy.Int(cast.ToInt(d.Get("restore_window_days"))),
+		CopyProtectionEnabled:   conversion.Pointer(d.Get("copy_protection_enabled").(bool)),
+		EncryptionAtRestEnabled: conversion.Pointer(d.Get("encryption_at_rest_enabled").(bool)),
+		PitEnabled:              conversion.Pointer(d.Get("pit_enabled").(bool)),
+		RestoreWindowDays:       conversion.Pointer(cast.ToInt(d.Get("restore_window_days"))),
```

Comment:
not sure why it's not similar to other places, e.g.   d.Get("restore_window_days").(int)

Reply:
🤷  I'm just replacing the pointer, there seem sto be a combination of using cast and `.int` all over